shared-scripts.sh

```diff
@@ -165,17 +166,16 @@ function _publishToLocalRegistry {
     loadCache repo $CODEBUILD_SRC_DIR
     loadCache .cache $HOME/.cache
 
-    source ./.circleci/local_publish_helpers.sh && startLocalRegistry "$CODEBUILD_SRC_DIR/.circleci/verdaccio.yaml"
+    source ./.circleci/local_publish_helpers_codebuild.sh && startLocalRegistry "$CODEBUILD_SRC_DIR/.circleci/verdaccio.yaml"
     setNpmRegistryUrlToLocal
-    export LOCAL_PUBLISH_TO_LATEST=true
-    ./.circleci/publish-codebuild.sh
+    ./.circleci/cb-publish-step-2-verdaccio.sh
     unsetNpmRegistryUrl
 
     echo Generate Change Log
     # Leaving this breadcrumb here "git reset --soft HEAD~1"
-    # we commented this out because the publish script is now checking out the current branch, and this started to fail as a result
+    # we commented this out previously because the publish script is now checking out the current branch, and this started to fail as a result
     # if we run into problems in the future, we should revisit this
-    # git reset --soft HEAD~1
+    git reset --soft HEAD~1
```

Comment:
What's the goal of doing a soft reset?

Reply:
if i remember correctly; its to remove the commit that is created during publish to verdaccio step so that it doesn't affect the unified changelog step

Reply:
this shouldn't be necessary. does CCI do this ?

Reply:
or maybe it does make some sense, should we unreset after step which is affected by this ?

Reply:
I see, this unified changelog script is designed to work with staged files:

```ts

 // This script is intended to be run after running `yarn publish-to-verdaccio` when all CHANGELOG file changes are staged but not committed
 // It will scan all staged CHANGELOG.md files and merge them into a single UNIFIED_CHANGELOG file
 ```